Dockerfile

```diff
@@ -1 +1,12 @@
+FROM alpine:latest
+
+LABEL description="Alpine Linux with Python 3"
+
```

Comment:
Add env var PYTHONPATH. It should contain python binary and path to python module in this repo.

Reply:
Do you mean the path to the copied python module inside the image, right?
Something like this ENV PYTHONPATH="$PYTHONPATH:/path-to-the-module"

Reply:
Exactly, but there can be only path to python binary and the module path.